src/Atmos/Model/turbulence.jl

```diff
@@ -321,13 +322,17 @@ end
 
 function turbulence_tensors(m::AnisoMinDiss, state::Vars, diffusive::Vars, aux::Vars, t::Real)
   FT = eltype(state)
+end
+function scaled_momentum_flux_tensor(m::AnisoMinDiss, ρν, S)
```

Comment:
@slifer50 : Hi Yassine, I am working on merging the CFL pull requests #618 & #667. Can I request you to provide a clarification on this revision. 

With this revision, the function turbulence_tensors, just returns FT = eltype(state). This seems to be called in the "courant_presets" as ν, τ = turbulence_tensors(m.turbulence, state, diffusive, aux, 0).  

- please advise if the diffusive_courant function has been approved from the physics/numerics point of view. 
- it may be a good idea to make any revisions to the Turbulence model in a different pull request.

Thanks.
cc: @simonbyrne, @kpamnany @akshaysridhar @bischtob